go.mod

```diff
@@ -9,8 +9,9 @@ require (
 	github.com/alecthomas/jsonschema v0.0.0-20200530073317-71f438968921
 	github.com/alicebob/miniredis/v2 v2.18.0
 	github.com/asaskevich/govalidator v0.0.0-20200108200545-475eaeb16496
-	github.com/btcsuite/btcutil v0.0.0-20190316010144-3ac1210f4b38
+	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
```

Comment:
A few days ago the `github.com/btcsuite/btcutil` library was moved under [btcutil](https://github.com/btcsuite/btcd/tree/master/btcutil) directory of the [btcd](https://github.com/btcsuite/btcd) repository. I'd use the new dependency path `github.com/btcsuite/btcd/btcutil` here instead

Reply:
I tried to address this but haven't done anything with go modules in years. It's likely I broke something https://github.com/brave-intl/bat-go/pull/1195/commits/a592867075f926dfddc81e8b853dd6ad5922e62b